radsat-sk/src/main.c

```diff
@@ -413,6 +431,227 @@ void MissionInitTask(void* parameters) {
+		uint8_t outputSize;
+		uint8_t wrappedMessage[] = {0x19, 0x21, 0x6f, 0xc6, 0x07, 0xe9, 0xf8, 0xe0, 0x63, 0x1b, 0x05, 0x0b, 0x03, 0x09, 0x0a};
+		uint8_t inputSize = 15; // Size of header + message
+		outputSize = messageUnwrap(wrappedMessage, inputSize, &result);
+		debugPrint("which_service: %d \n", (uint8_t) result.which_service);
+		debugPrint("message size: %d \n", outputSize);
+		if (result.which_service == radsat_message_TelecommandMessage_tag){
+			debugPrint("Successfully read that this message is a telecommand message.\n");
+			// obtain the specific telecommand
+			uint8_t telecommand = (uint8_t) result.TelecommandMessage.which_message;
+			if(telecommand == telecommand_message_BeginPass_tag){
+				debugPrint("Successfully read that this message is a Begin Pass message. \n");
+				debugPrint("passLength value read as: %d. Should be 11.\n", result.TelecommandMessage.BeginPass.passLength);
+				debugPrint("output size is: %d\n", outputSize);
+			}
+		}
+		*/
+
+		//test two
+/*
+	radsat_message result = {0};
+	uint8_t outputSize;
+	uint8_t wrappedMessage[] = {0x19, 0x21, 0x0a, 0x84, 0x3d, 0x04, 0xfe, 0xe0, 0x63, 0x13, 0x3b, 0x1b, 0x39, 0x09, 0x04, 0x13, 0x19, 0x0c, 0x9b, 0x98, 0xa8, 0x41, 0x14, 0x9b, 0x98, 0xa8, 0x41, 0x1c, 0x9b, 0x98, 0xa8, 0x41, 0x24, 0x9b, 0x98, 0xa8, 0x41, 0x29, 0x04, 0x31, 0x04, 0x1b, 0x0d, 0x09, 0x04, 0x11, 0x04, 0x19, 0x04, 0x21, 0x04, 0x29, 0x04, 0x31, 0x04, 0x23, 0x0d, 0x09, 0x04, 0x11, 0x04, 0x19, 0x04, 0x21, 0x04, 0x29, 0x04, 0x31, 0x04};
+	uint8_t inputSize = 69; // Size of header + message
+	outputSize = messageUnwrap(wrappedMessage, inputSize, &result);
+	debugPrint("which_service: %d \n", (uint8_t) result.which_service);
+	debugPrint("message size: %d \n", outputSize);
+	if (result.which_service == radsat_message_FileTransferMessage_tag){
+		debugPrint("Successfully read that this message is a FileTransfer message.\n");
+		// obtain the specific telecommand
+		uint8_t telecommand = (uint8_t) result.FileTransferMessage.which_message;
+		if(telecommand == file_transfer_message_CameraTelemetry_tag){
+			debugPrint("Successfully read that this message is a Camera Telemetry message. \n");
+			debugPrint("uptime value read as: %d. Should be 5.\n", result.FileTransferMessage.CameraTelemetry.uptime);
+			debugPrint("current 3v3 from powerTelemetry submessage: %f. Should be 5.3.\n", result.FileTransferMessage.CameraTelemetry.powerTelemetry.current_3V3);
+			debugPrint("detectionThreshold from cameraTwoTelemetry submessage: %d. Should be 5.\n", result.FileTransferMessage.CameraTelemetry.cameraOneTelemetry.detectionThreshold);
+			debugPrint("output size is: %d\n", outputSize);
+		}
+	}
+	*/
+
+
+
+
+
+
+	//vTaskDelay(1000);
+
+
+	//printf("start here\n");
+
+
+	//uint8_t mess[200] = {0};
+	//for (int i= 0; i < 200; i++){
+	//	mess[i] = 0b10101010;
+	//}
+	//while (1){
+	//	error = IsisTrxvu_tcSendAX25DefClSign(0, mess, 200, 0);
+	//	if (error) debugPrint("error = %d\n",error);
+	//	vTaskDelay(100);
+	//}
```

Comment:
Remove this but save the trx stuff and keep the testing this would be good for dosimeter testing and some other stuff.